### DISCUSSION TITLE
Micropython Docs site down

### DISCUSSION THREAD

**Discussion by bromeara** [micropython/micropython#12456] [General]
The url https://docs.micropython.org/en/latest currently shows the error page 'MicroPython Documentation Not Found!' going to a specific page url does not fix it. It seems as thought it was online as recently as 9/13/2023 according to the wayback machine https://web.archive.org/web/20230913015207/https://docs.micropython.org/en/latest . Ive tried on both my local network and mobile network and got the same result. I didn't see an announcement for any maintenance activities is this occurring for anyone else?

**Comment by bromeara** [comment]
Also the link from 09/11/2023 looks more normal than 09/13/2023 Im not sure if the update on 09/12/2023 broke something. https://web.archive.org/web/20230911065249/https://docs.micropython.org/en/latest/

**Reply by agarthetiger** [reply]
I'm seeing the same issue. I've just been searching for "micropython machine reset" and all the links to this site from google are showing the same, as is the link to the docs from the bare micropython.org site, which I can get to ok.

<img width="898" alt="Screenshot 2023-09-15 at 20 56 28" src="https://github.com/micropython/micropython/assets/1242023/add21433-52e8-4178-af2d-22ce19b80265">

**Comment by jimmo** [comment]
Damien knows about it, but it's Saturday morning here so not sure when he'll be able to look at it.

In the meantime you can use the docs for the previous release:
http://docs.micropython.org/en/v1.20.0/ (fixed)

**Reply by jimmo** [reply]
I meant http://docs.micropython.org/en/v1.20.0/ sorry

**Comment by jimmo** [comment]
Docs are back online now! sorry about that!

**Reply by agarthetiger** [reply]
No worries, thank you 👍🏼

**Comment by sophiedegran** [comment]
Sorry docs are online but can't search anything and versions/download do nothing...

**Reply by dpgeorge** [reply]
Should be fixed by 9dd40d1b607dae7b943d57a61e83191687a5683e

**Comment by sophiedegran** [comment]
In progress! Search is ok, Version and download are working except the download latest version
-T

**Comment by Josverl** [comment]
@dpgeorge,
I see build errors for the documentation in a [PR's checks](https://github.com/micropython/micropython/actions/runs/6275035002/job/17041618910?pr=12500)
```
Could not import extension sphinxcontrib.jquery (exception: No module named 'sphinxcontrib.jquery')
```
actually in all docs builds since : https://github.com/micropython/micropython/actions/workflows/docs.yml

seems like the sphinx modules are not updated in all locations

**Reply by robert-hh** [reply]
And me. See #12498

**Comment by apiep** [comment]
The documentation are missing again https://docs.micropython.org/en/latest

**Comment by GitHubsSilverBullet** [comment]
I'd strongly suggest to build the HTML doc (and for those not comfortable with that to  _tar.bz2_ it for download).
Nothing is easier than to install a local lightweight Web-Server on a Linux system and put the stuff there.
Resulting in lightning fast access even without Internet connectivity.
And one can add own pages with documentation for homegrown extensions.

PS. And yes, the _sphinxcontrib.jquery_ truly is a pain in the…
